src/Main_Panel.cpp

```diff
@@ -1192,7 +1192,11 @@ void main_panel::texture_progbar_end(wxAsyncProgressEndEvt &event)
 void main_panel::SignalModelChange(std::string filename, bool skipProgdlg) 
 {
 	UseThreadedProgBar = !skipProgdlg;
-	wxSetWorkingDirectory(wxString(filename.c_str(), wxConvUTF8).BeforeLast('\\'));
+	wxString tdir;
+	wxFileName::SplitPath(wxString(filename.c_str(), wxConvUTF8), &tdir, nullptr, nullptr);
```

Comment:
not sure what the benefit is to using a wxDir? Doesn't seem to be any simpler to use that when getting the dir from a filename & testing that the dir exists.
http://docs.wxwidgets.org/3.0/classwx_dir.html